Documentation/get-dotnetcore-windows-dnvm.md

```diff
@@ -0,0 +1,102 @@
+
+	@powershell -NoProfile -ExecutionPolicy unrestricted -Command "&{$Branch='dev';iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/aspnet/Home/dev/dnvminstall.ps1'))}"
+
+
+You can see the currently installed DNX versions with `dnvm list`, which will display an empty set of installed runtimes.
+
+	C:\coreclr-demo>dnvm list
+
+Installing a .NET Core DNX
+==========================
+
+It's easy to install the latest .NET Core-based DNX, using the ```dnvm install``` command. 
+
+	C:\coreclr-demo>dnvm install -r coreclr latest
+
+This will install the 32-bit version of .NET Core. If you want the 64-bit version, you can specific processor architecture.
+
+	C:\coreclr-demo>dnvm install -r coreclr -arch x64 latest
+
+You can see the currently installed DNX versions with `dnvm list`.
+
+	C:\coreclr-demo>dnvm list
+
+	Active Version           Runtime Architecture Location
+	------ -------           ------- ------------ --------
+	  *    1.0.0-beta5-11596 coreclr x64          C:\Users\rlander\.dnx\runtimes
+	       1.0.0-beta5-11596 coreclr x86          C:\Users\rlander\.dnx\runtimes
+
+You can choose which of these DNXs you want to use with `dnvm use`, with similar arguments.
+
+	C:\coreclr-demo>dnvm use -r coreclr -arch x86 1.0.0-beta5-11596
+	Adding C:\Users\rlander\.dnx\runtimes\dnx-coreclr-win-x86.1.0.0-beta5-11596\bin
+	to process PATH
+
+	C:\coreclr-demo>dnvm list
+
+	Active Version           Runtime Architecture Location
+	------ -------           ------- ------------ --------
+	       1.0.0-beta5-11596 coreclr x64          C:\Users\rlander\.dnx\runtimes
+	  *    1.0.0-beta5-11596 coreclr x86          C:\Users\rlander\.dnx\runtimes
+
+Write your App
+==============
+
+You need a Hello World application to run. You can write your own, if you'd like. Here's a very simple one:
+
+```csharp
+using System;
+
+public class Program
+{
+    public static void Main (string[] args)
+    {
+        Console.WriteLine("Hello, Windows");
+        Console.WriteLine("Love from CoreCLR.");
+    }   
+} 
+```
+
+Some people on the .NET Core team are partial to a demo console app on corefxlab repo which will print a picture for you. Download the [corefxlab demo](https://raw.githubusercontent.com/dotnet/corefxlab/master/demos/CoreClrConsoleApplications/HelloWorld/HelloWorld.cs) to ```C:\coreclr-demo```.
+
+You need a `project.json` that matches your app. Use this one. It will work for both of the apps provided/referenced above. Save the project.json beside your app.
+
+```
+{
+    "version": "1.0.0-*",
+    "dependencies": {
+    },
+    "frameworks" : {
+        "dnx451" : { },
+        "dnxcore50" : {
+            "dependencies": {
+                "System.Console": "4.0.0-beta-*"
+            }
+        }
+    }
+}
+```
```

Comment:
I guess in this case you don't need to dnu restore because System.Console ships as part of DNX?  It might be good to call out the restore step anyway since that's the typical flow.